release.sh

```diff
@@ -69,7 +69,12 @@ mkdir -p ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/nfs
 
 cp -pr bootmnt/* ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/tftp/
 cp -pr rootmnt/* ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/nfs/
-cp -pr ROOTFS/media/rpi/sage-utils/dhcp-pxe/* ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/
+cp -pr ROOTFS/media/rpi/sage-utils/dhcp-pxe/* ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe
+zcat ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/tftp/vmlinuz > ${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/tftp/vmlinux
+
+wget https://github.com/rancher/k3s/releases/download/v1.20.0+k3s2/k3s-arm64
```

Comment:
In addition to getting the binary can you also curl the script itself (from https://get.k3s.io/) and put it in `${BASEDIR}/media/rpi/sage-utils/dhcp-pxe/nfs/etc/installK3S.sh` instead of committing a static version of the file?

I can see a reason to commit a static version of the k3s installer shell script.... getting it dynamically has the advantage of not needing to commit it to our code repo.  But we would not get any bug fixes dynamically.  We would also not get any new bugs....

Basically, if you feel that since u are getting a specific version of k3s (v1.20.0) that you also want a static version of the installer script then I would be okay with keeping it checked in.

Reply:
Yeah, I think it's best to keep this version of the installer script checked in since we don't know how future installer scripts may look, they could rename some parameters that would make this not work